docs_user/modules/proc_using-new-subnet-ranges.adoc

```diff
@@ -127,25 +101,20 @@ Finally, during the data plane adoption, you will have to take care of several a
         fixedIP: 192.168.122.100
         name: ctlplane
         subnetName: subnet1
-```
-
-- expand SSH range for the firewall configuration to include both subnets:
+----
 
-```yaml
+. Expand the SSH range for the firewall configuration to include both subnets:
++
+----
         edpm_sshd_allowed_ranges:
         - 192.168.122.0/24
         - 172.31.0.0/24
-```
-
-This is to allow SSH access from the new subnet to the adopted nodes as well as
-the old one.
-
----
-
-Since you are applying new network configuration to the nodes, consider also
-setting `edpm_network_config_update: true` to enforce the changes.
+----
++
+This provides SSH access from the new subnet to the adopted nodes as well as
+the old nodes.
 
----
+. Optional: Set `edpm_network_config_update: true` to enforce the changes that you are applying to the nodes:
```

Comment:
Where do customers set the edpm_network_config_update: true? Can we please provide a code example in this step?

Reply:
I can find an example, but first I'd like to check with e.g. @slagle or @dprince if we really need to add this setting when we do the initial adoption. Guys, is it really needed?